packages/components/src/modifiers/hds-code-editor.ts

```diff
@@ -0,0 +1,132 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import { assert } from '@ember/debug';
+import Modifier from 'ember-modifier';
+import {
+  lineNumbers,
+  highlightActiveLineGutter,
+  highlightSpecialChars,
+  drawSelection,
+  highlightActiveLine,
+  EditorView,
+  keymap,
+} from '@codemirror/view';
+import { EditorState, type Extension } from '@codemirror/state';
```

Comment:
suggestion: investigate if we can defer importing these until setup.

Reply:
Good call, done in the latest.